test/sharness.d/0020-create-test-repository.sh

```diff
@@ -46,7 +46,14 @@ create_test_repository () {
 			https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.31.1.tar.gz &&
 		gunzip "${SHARNESS_TEST_SRCDIR}/git.tar.gz" &&
 		wget -O "${SHARNESS_TEST_SRCDIR}/git.tar.sign" \
-			https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.31.1.tar.sign &&
+			https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.31.1.tar.sign
+		if test $? -ne 0
+		then
+			echo >&2 "ERROR: fail to download or unzip git.tar.gz"
+			# release the lock
+			rm -f "${PO_HELPER_TEST_REPOSITORY}.lock"
+			exit 1
+		fi
 		gpg --verify "${SHARNESS_TEST_SRCDIR}/git.tar.sign"
```

Comment:
Can we move downloading *.sign file just above this line, and try to verify git package.
We should not quit if fail to verify git package, because user or CI system may not install Junio's pubkeys.